pkg/apimachinery/registered/registered.go

```diff
@@ -111,6 +111,7 @@ var (
 	IsRegisteredVersion           = DefaultAPIRegistrationManager.IsRegisteredVersion
 	IsRegistered                  = DefaultAPIRegistrationManager.IsRegistered
 	Group                         = DefaultAPIRegistrationManager.Group
+	EnabledGroups                 = DefaultAPIRegistrationManager.EnabledGroups
```

Comment:
This looks reasonable to me, but cc @lavalamp since I don't know his plan for this package.